recipes/textual/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "textual" %}
+{% set version = "0.1.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 80ebb32f67489ab88c1eeabca23dc33beac3cef7e978a0acac4df34fdad047ba
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py<37]
+
+requirements:
+  host:
+    - python
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python
+    - rich >=10.7.0,<11
+    - typing-extensions >=3.10.0,<4  # [py<38]
```

If making noarch, just change to an unconditional dependency:
```suggestion
    - typing-extensions >=3.10.0,<4
```